crates/bench/benches/subscription.rs

```diff
@@ -18,7 +18,10 @@ fn create_table_location(db: &RelationalDB) -> Result<TableId, DBError> {
         ("z", AlgebraicType::I32),
         ("dimension", AlgebraicType::U32),
     ];
-    db.create_table_for_test_multi_column("location", schema, col_list![2, 3, 4])
+    let indexes = &[(0.into(), "entity_id"), (2.into(), "owner_entity_id")];
```

Comment:
I think you meant 0 and 1 for entity_id and chunk_index.

Reply:
Correct